src/main/java/com/minecolonies/api/colony/buildings/IBuilding.java

```diff
@@ -319,7 +319,12 @@ <R> ImmutableList<IRequest<? extends R>> getOpenRequestsOfType(
 
     boolean hasCitizenCompletedRequestsToPickup(@NotNull ICitizenData data);
 
-    Collection<IRequest<?>> getCompletedRequests(@NotNull ICitizenData data);
+    /**
+     * Get completed requests for citizen or building (citizen independent, so if data is null)
+     * @param data the citizen data (or null).
+     * @return a collection of request.
+     */
+    Collection<IRequest<?>> getCompletedRequestsOrCitizenOrBuilding(@Nullable ICitizenData data);
```

Comment:
```suggestion
    Collection<IRequest<?>> getCompletedRequestsOfCitizenOrBuilding(@Nullable ICitizenData data);
```

This looks like a typo